appveyor.yml

```diff
@@ -0,0 +1,18 @@
+# scripts that run after cloning repository
+install:
+  - if "%APPVEYOR_REPO_BRANCH%" EQU "master" if not defined APPVEYOR_PULL_REQUEST_NUMBER set MASTER_BUILD=1
```

Comment:
Ah yes, indeed that may be a problem. I'll take a look at how to detect if it's running on a fork.